__tests__/review/displayHelpMessage.test.ts

```diff
@@ -0,0 +1,30 @@
+import { HTTP_STATUS_OK } from '@/constants';
+import { waitFor } from '@root/__tests__/utils/waitFor';
+import { fetch } from '../utils/fetch';
+import { getSlackHeaders } from '../utils/getSlackHeaders';
+
+describe('help', () => {
+  it('help message should be displayed', async () => {
+    // Given
+    const channelId = 'channelId';
+    const userId = 'userId';
+    const body = {
+      channel_id: channelId,
+      text: 'help',
+      user_id: userId,
+    };
+
+    // When
+    const response = await fetch('/api/v1/homer/command', {
+      body,
+      headers: getSlackHeaders(body),
+    });
+
+    // Then
+    expect(response.status).toEqual(HTTP_STATUS_OK);
+
+    await waitFor(() => {
+      // expect(slackBotWebClient.chat.postEphemeral).toHaveBeenCalledTimes(1);
+    });
```

Comment:
After looking at the code, here are my thoughts:

When calling the command with the text 'help', the [commandRequestHandler](https://github.com/ManoManoTech/homer/blob/main/src/core/requestHandlers/commandRequestHandler.ts#L29) goes to the default switch and sends directly to Slack the JSON built using [buildHelpMessage](https://github.com/ManoManoTech/homer/blob/main/src/core/viewBuilders/buildHelpMessage.ts#L4) function, instead of using the Slackbot webclient's `postEphemeral` function. So instead of the `await waitFor...` statements, you can do the following:

```suggestion   
    const slackMessage = await response.json(); 

    // Then
    expect(response.status).toEqual(HTTP_STATUS_OK);
    expect(slackMessage.text).toContain('Here are the available commands:');
```
Here is a simple workflow of what is happening:

/homer help -> route to commandRequestHandler -> switch default -> buildHelpMessage -> send help message to Slack

If it is not clear, do not hesitate to ping me and we can review it together.

Reply:
Suggestion: add `expects` for both Slack channel name and IDs.

Reply:
Done!  
Channel ID cannot be checked but channel name is now checked. Thanks for the help @pfongkye